src/llm/aqueduct_llm/dolly_v2_7b.py

```diff
@@ -0,0 +1,106 @@
+import time
+from typing import List, Union
+
+import torch
+from aqueduct_llm.utils.dolly_instruct_pipeline import InstructionTextGenerationPipeline
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+default_do_sample = True
+default_max_new_tokens = 256
+default_top_p = 0.92
+default_top_k = 0
+
+
+class Config:
+    def __init__(
+        self,
+        do_sample: bool,
+        max_new_tokens: int,
+        top_p: float,
+        top_k: int,
+    ):
+        self.model_path = "databricks/dolly-v2-7b"
+        self.do_sample = do_sample
+        self.max_new_tokens = max_new_tokens
+        self.top_p = top_p
+        self.top_k = top_k
+
+    def describe(self) -> str:
+        print("Running Dolly V2 7B with the following config:")
+        attrs = {
+            "do_sample": self.do_sample,
+            "max_new_tokens": self.max_new_tokens,
+            "top_p": self.top_p,
+            "top_k": self.top_k,
+        }
+        print("\n".join([f"{attr}: {value}" for attr, value in attrs.items()]))
+
+
+def generate(
+    messages: Union[str, List[str]],
+    do_sample: bool = default_do_sample,
+    max_new_tokens: int = default_max_new_tokens,
+    top_p: float = default_top_p,
+    top_k: int = default_top_k,
+) -> Union[str, List[str]]:
+    """Invoke the Dolly V2 7B model to generate responses.
+
+    The model requires a GPU and at least 16GB of RAM.
```

Comment:
Just a note to also add this to the documentation with the relevant code snippet example to run this on Aqueduct.